apps/ledger-live-desktop/src/renderer/reducers/UI.ts

```diff
@@ -46,6 +46,8 @@ export type UIState = {
     isOpen: boolean;
     payload?: PlatformAppDrawers | null;
   };
+  isMemoTagBoxVisible: boolean;
+  forceAutoFocusOnMemoField?: boolean;
```

Comment:
you should add it in INITIAL_STATE also no?

Reply:
while it's an optional + boolean state property -> it will have an automatic falsy initial state . 
But yeah why not make it explicit.

Reply:
done ! ☑️